tests/Converters/CodeConverterTest.php

```diff
@@ -188,6 +188,27 @@ public function test_non_test_method()
         ->not->toContain('$this->thisIsNotATest()');
 });
 
+it('convert non test static method calls', function () {
+    $code = '<?php
+        class MyTest {
+            protected static function thisIsNotATest() {}
+
+            public function test_non_test_method()
+            {
+                self::thisIsNotATest();
+            }
+        }
+    ';
+
+    $convertedCode = codeConverter()->convert($code);
+
+    expect($convertedCode)
+        ->toContain('function thisIsNotATest()')
+        ->not->toContain('protected function thisIsNotATest()')
```

Comment:
I think the keyword `private` is missing here.

```suggestion
        ->not->toContain('protected static function thisIsNotATest()')
```

Reply:
Yes, you're right, it was missing.